superset/connectors/base/models.py

```diff
@@ -51,6 +52,11 @@
 ]
 
 
+class DatasourceKind(Enum):
```

Comment:
Most enums i've seen in python have been in SHOUT_CASE, should we be consistent here?

Also, I think if you have this extend `str` as well, then instead of returning `DatasourceKind.virtual.value` below you can return `str(DatasourceKind.virtual)`. see https://github.com/apache/incubator-superset/blob/7f6dbf838e4e527e640a002ce20bf5da1abf4a98/superset/errors.py#L24